src/plugins/intel_gpu/src/graph/impls/ocl/broadcast.cpp

```diff
@@ -62,7 +62,12 @@ struct broadcast_impl : typed_primitive_impl_ocl<broadcast> {
         auto output_rank = output_pshape.size();
 
         if (primitive->axes_mapping.empty()) {
-            input_pshape = extend_shape_to_rank_from_begin(input_pshape, output_rank);
+            bool use_new_shape_infer = impl_params.prog->get_config().get_property(ov::intel_gpu::allow_new_shape_infer);
```

Comment:
@Lyamin-Roman
Could you review it? I'm not sure but is it intended to always use 'extend_shape_to_rank_from_begin(..)' here regardless static and dynamic?
static broadcast has the issue since https://github.com/openvinotoolkit/openvino/pull/16166